.github/workflows/build.yml

```diff
@@ -0,0 +1,149 @@
+name: build
+on: [push, pull_request]
+jobs:
+  checks:
+    runs-on: ubuntu-latest
+    container: "renaissancebench/buildenv:openjdk8"
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Cache SBT
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.ivy2/cache
```

Comment:
If this is caching, we might want to also include `$HOME/.cache/coursier`

Reply:
Added, thanks!